Remotion/Web/Development.Analyzers/RMWEB0001_WebStringAnalyzer.cs

```diff
@@ -87,14 +89,45 @@ public override void Initialize (AnalysisContext context)
                 stringBuilderAppendMethodSymbol,
                 htmlTextWriterWriteMethodSymbol,
                 stringJoinMethodSymbols,
-                stringFormatMethodSymbols);
+                stringFormatMethodSymbols,
+                stringConcatMethodSymbols);
 
+            compilationContext.RegisterSyntaxNodeAction(
+                ctx => AnalyzeAddExpression (ctx, symbolContext),
+                ImmutableArray.Create (SyntaxKind.AddExpression));
             compilationContext.RegisterSyntaxNodeAction (
                 ctx => AnalyzeInvocationExpression (ctx, symbolContext),
                 ImmutableArray.Create (SyntaxKind.InvocationExpression));
           });
     }
 
+    private void AnalyzeAddExpression (SyntaxNodeAnalysisContext ctx, SymbolContext symbolContext)
+    {
+      var node = (BinaryExpressionSyntax) ctx.Node;
```

Comment:
Doesn't this also cover `==` operators etc (there's no test for the actual operator in the check)?

Reply:
The action is registered for `SyntaxKind.AddExpression`, which filters the operator beforehand.

Reply:
ah, thanks, there it is!